plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -3484,6 +3483,19 @@ eosio::chain::backing_store_type read_only::get_backing_store() const {
 }
 
 } // namespace chain_apis
+
+fc::variant chain_plugin::get_entire_trx_trace(const std::variant<std::shared_ptr<transaction>, transaction_trace_ptr>& obj )const {
+
+    fc::variant pretty_output;
+    auto ro_api = get_read_only_api();
+    try {
+        abi_serializer::to_variant(obj, pretty_output,
+                                   chain_apis::make_resolver(&ro_api, abi_serializer::create_yield_function(get_abi_serializer_max_time())),
+                                   abi_serializer::create_yield_function(get_abi_serializer_max_time()));
+    }EOS_RETHROW_EXCEPTIONS(chain::packed_transaction_type_exception, "Invalid packed transaction trace")
```

Comment:
I think you might have misunderstood my previous PR comments. This method should never throw. Instead of rethrow it should just assign directly to `pretty_output`.

Reply:
Also I don't think abi_serializer will actually work as is like this by passing in a `std::variant`. You could pull out of the particular type and pass that in OR have the method take two args instead of a variant OR have two diff methods OR make it a template.

Reply:
Fixed